base/src/test/java/io/spine/validate/RangeConstraintTest.java

```diff
@@ -78,4 +92,13 @@ void throwOnMalformedRanges(String badRange) {
                 Arguments.of("[3...5]")
         );
     }
+
+    private static Stream<Arguments> rangeCombinationsFor(Number left, Number right) {
+        return Stream.of(
+                Arguments.of(format("[%s..%s]", left, right)),
```

Comment:
Please refactor this code so that we don't repeat formatting. You have enum with the ends, don't you?